includes/class-dispatcher.php

```diff
@@ -125,7 +125,7 @@ public static function process_outbox( $id ) {
 	 *
 	 * @return array|void The next batch of followers to process, or void if done.
 	 */
-	public static function send_to_followers( $outbox_item_id, $batch_size = 50, $offset = 0 ) {
+	public static function send_to_followers( $outbox_item_id, $batch_size = ACTIVITYPUB_OUTBOX_PROCESSING_BATCH_SIZE, $offset = 0 ) {
```

Comment:
You also have to change the phpdoc to `Default ACTIVITYPUB_OUTBOX_PROCESSING_BATCH_SIZE.`.

Reply:
Fixed in [2e16efd](https://github.com/Automattic/wordpress-activitypub/pull/1514/commits/2e16efdf34a80ffdff76e27796ebd27f404ae5e9)